### PR TITLE
Added 015_3Sum.java

### PR DESCRIPTION
**Pull request by green-study**
Added 015_3Sum.java

### CODE REVIEW

File: java/015_3Sum.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                } else {
+                    //if sum < 0, left++;
+                    left++;
+                }
+            }
+        }
+        
+        return result; //return result list
+    }
+}
```
Comment (qiyuangong) [inline java/015_3Sum.java]: Add an empty line at the end. :)

File: java/015_3Sum.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        
+        //sorting nums
+        Arrays.sort(nums);
+        
+        for (int i = 0; i < nums.length - 2; i++) {
+
+            int left = i + 1;
+            int right = nums.length - 1;
+
+            if (i > 0 && nums[i] == nums[i-1]){
```
Comment (qiyuangong) [inline java/015_3Sum.java]: Code style: Space before `{`

File: java/015_3Sum.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
+//015. 3Sum
```
Comment (qiyuangong) [inline java/015_3Sum.java:1]: Other LGTM :)

**Comment by green-study** [comment]
The modification has been completed.

**Review by qiyuangong — APPROVED** [review]
LGTM

**Pull request merged**